src/main/java/karen/command/ListCommand.java

```diff
@@ -0,0 +1,17 @@
+package karen.command;
+
+import karen.Storage;
+import karen.Ui;
+
+/**
+ * To list objects currently in Storage
+ */
+public class ListCommand extends Command {
+    public ListCommand() {
+    }
+
+    @Override
```

Comment:
For Overidden methods, you can use {@inheritDoc} on your Java Doc method header.

Reply:
Good catch - overlooked on this. Thanks for the suggestion!